tbump.toml

```diff
@@ -0,0 +1,61 @@
+github_url = "https://github.com/scikit-hep/pyhf/"
+
+[version]
+current = "0.6.3"
+
+# Example of a semver regexp.
+# Make sure this matches current_version before
+# using tbump
+regex = '''
+  (?P<major>\d+)
+  \.
+  (?P<minor>\d+)
+  \.
+  (?P<patch>\d+)
+  (rc
+    (?P<candidate>\d+)
+  )?
+  '''
+
+[git]
+# The current version will get updated when tbump is run
+message_template = "Bump version: 0.6.3 → {new_version}"
+tag_template = "v{new_version}"
+
+# For each file to patch, add a [[file]] config
+# section containing the path of the file, relative to the
+# tbump.toml location.
+[[file]]
+src = "tbump.toml"
+# Restrict search to make it explicit why tbump.toml
+# is even included as a file to bump, as it will get
+# its version.current attribute bumped anyway.
+search = "Bump version: {current_version} → "
+
+[[file]]
+src = "src/pyhf/utils.py"
+# Guard SCHEMA_VERSION
+# This search is just identifying the line to restrict the
+# regex to, but all matches in the line will get bumped.
+search = "pyhf: v{current_version}"
```

Comment:
So this search term will restrict to this line, but hit all the versions in it,

https://github.com/scikit-hep/pyhf/blob/2533bd9b391e00614ad5e33781a4a0977f588263/src/pyhf/utils.py#L135

and so won't ever match

https://github.com/scikit-hep/pyhf/blob/2533bd9b391e00614ad5e33781a4a0977f588263/src/pyhf/utils.py#L13